migration-guides/v6.0.0.md

```diff
@@ -1,24 +1,28 @@
+# v6.0.0 Migration Guide
 This version contains the following breaking changes:
 
 1. `<Select />` now has a `placeholder` prop by default ("Select")
 1. `<Button />` and `<SubmitButton />` now use `aria-disabled` instead of `disabled`
 1. The `style` prop on `<Button />` and `<SubmitButton />` has been renamed to `variant`
 1. Labels for `<CheckboxGroup />` and `<RadioGroup />` now use a `<legend>` html tag rather than a `<label>`
 1. The `hideLabel` prop on `<RangeInput />` has been renamed to `hideRangeValue`
+1. `<CloudinaryFileInput />` and `<FileInput />` components now store an object (or array of objects) representing the file data as opposed to a string
+1. `<CloudinaryFileInput />` and `<FileInput />` components no longer accept an `onLoad` prop
+1. The `previewComponent` for a file input no longer receives a `file` prop and `value` is a file object (or array of file objects) as opposed to a string
```

Comment:
Does it make it easier to have the same interface for the `value` to always be an array and just an array of one object in the non-multi file support scenario?

Reply:
That would match the native [FileList](https://developer.mozilla.org/en-US/docs/Web/API/FileList) implementation

Reply:
Sorry! The edit didn't come through email, but yeah, I like the simplification (just always being an array). Taking a look at these updates now.